src/Admin/AdminConsole/Controllers/ProvidersController.cs

```diff
@@ -290,25 +291,44 @@ public async Task<IActionResult> Edit(Guid id, ProviderEditModel model)
 
         var providerPlans = await _providerPlanRepository.GetByProviderId(id);
 
-        if (providerPlans.Count == 0)
+        // Billable providers should always have plans at this point in time?
+        if (provider.IsBillable() && providerPlans.Count == 0)
```

Comment:
@amorask-bitwarden Not sure whether this was okay, but billable providers should have provider plans at this point.